Owin.Security.Providers/Backlog/BacklogAuthenticationHandler.cs

```diff
@@ -72,9 +73,12 @@ public BacklogAuthenticationHandler(HttpClient httpClient, ILogger logger)
                 body.Add(new KeyValuePair<string, string>("client_secret", Options.ClientSecret));
 
                 // Get token
-                httpClient.DefaultRequestHeaders.Authorization = null;
-                HttpResponseMessage tokenResponse =
-                    await httpClient.PostAsync(Options.TokenEndpoint, new FormUrlEncodedContent(body));
+                var tokenRequest = new HttpRequestMessage(HttpMethod.Post, Options.TokenEndpoint);
+                tokenRequest.Headers.Accept.Add(new MediaTypeWithQualityHeaderValue("application/json"));
+                tokenRequest.Content = new FormUrlEncodedContent(body);
+
+                HttpResponseMessage tokenResponse = await httpClient.SendAsync(tokenRequest);
```

Comment:
nit: you could use `await httpClient.SendAsync(tokenRequest, Request.CallCancelled);`, but that's a detail.